pslogin/src/main/scala/WorldSessionActor.scala

```diff
@@ -8898,6 +9276,71 @@ class WorldSessionActor extends Actor with MDCContextAware {
     }
   }
 
+  def SwapSquadUIElements(squad : Squad, fromIndex : Int, toIndex : Int) : Unit = {
```

Comment:
Heavy business logic functions like these are excellent candidates for moving to a `private[WorldSessionActor]` separate file, if possible.

Reply:
I need to pass the `WorldSessionActor.squadUI` `LongMap` object?

Reply:
we dont need to worry about it for this patch, but these kind of functions are good to move out in the future. how we go about that, still needs thinking